JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -368,12 +401,20 @@ module Microsoft.ApplicationInsights {
         public _xhrReadyStateChange(xhr: XMLHttpRequest, payload: string[], countOfItemsInPayload: number) {
             if (xhr.readyState === 4) {
                 if ((xhr.status < 200 || xhr.status >= 300) && xhr.status !== 0) {
-                    this._onError(payload, xhr.responseText || xhr.response || "");
+                    if (!this._config.disableRetry() && this._isRetriable(xhr.status)) {
```

Comment:
When using config.disableRetry() it reads like a command which will disable retrying instead of a getter for a flag.
I think it would be cleared if it were renamed to something like "isRetryDisabled" so that it is clear it is a check and not a command.